analyzer/src/main/kotlin/managers/Bower.kt

```diff
@@ -19,20 +19,265 @@
 
 package com.here.ort.analyzer.managers
 
-import com.here.ort.analyzer.PackageManager
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.SerializationFeature
+
 import com.here.ort.analyzer.AbstractPackageManagerFactory
+import com.here.ort.analyzer.PackageManager
+import com.here.ort.downloader.VersionControlSystem
+import com.here.ort.model.Identifier
+import com.here.ort.model.Package
+import com.here.ort.model.PackageReference
+import com.here.ort.model.Project
+import com.here.ort.model.ProjectAnalyzerResult
+import com.here.ort.model.RemoteArtifact
+import com.here.ort.model.Scope
+import com.here.ort.model.VcsInfo
 import com.here.ort.model.config.AnalyzerConfiguration
 import com.here.ort.model.config.RepositoryConfiguration
+import com.here.ort.model.jsonMapper
+
+import com.here.ort.utils.CommandLineTool
+import com.here.ort.utils.OS
+import com.here.ort.utils.fieldNamesOrEmpty
+import com.here.ort.utils.log
+import com.here.ort.utils.safeDeleteRecursively
+import com.here.ort.utils.textValueOrEmpty
+
+import com.vdurmont.semver4j.Requirement
+
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.StandardCopyOption
+import java.util.SortedSet
+import java.util.Stack
 
 /**
  * The Bower package manager for JavaScript, see https://bower.io/.
  */
 class Bower(analyzerConfig: AnalyzerConfiguration, repoConfig: RepositoryConfiguration) :
-        PackageManager(analyzerConfig, repoConfig) {
+        PackageManager(analyzerConfig, repoConfig), CommandLineTool {
+
     class Factory : AbstractPackageManagerFactory<Bower>() {
         override val globsForDefinitionFiles = listOf("bower.json")
 
         override fun create(analyzerConfig: AnalyzerConfiguration, repoConfig: RepositoryConfiguration) =
                 Bower(analyzerConfig, repoConfig)
     }
+
+    companion object {
+        /* We do not actually depend on any features specific to Bower version 1.8.2, but we still want to
+          stick to fixed versions to be sure to get consistent results. */
+        private const val REQUIRED_BOWER_VERSION = "1.8.2"
```

Comment:
Should we allow at least `1.8.*`? You could check `NPM.kt` for an example for version ranges.

Reply:
Will do in a commit on top along with some testing.